javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,49 @@
+const width = 6;
+const height = 12;
+
+const area = width * height;
+
+console.log('value of area is:', area);
+console.log('typeof area is:', typeof area);
+
+const bill = 150;
+const payment = 500;
+
+const change = bill - payment;
+
+console.log('value of change is:', change);
+console.log('typeof change is:', typeof change);
+
+const quizzes = 50;
+const midterm =  150; 
+const final = 200;
+
+const grade = (quizzes + midterm + final) / 3;
+
+console.log('value of grade is:', grade);
+console.log('typeof grade is:', typeof grade);
+
+const firstName = "Dennis";
+const lastName = "Nguyen";
+let fullName = firstName + ' ' + lastName;
+
+console.log('value of fullName is:', fullName);
+console.log('typeof fullName is:', typeof fullName);
+
+const pH = 5;
+const isAcidic = pH < 7; 
+
+console.log('value of isAcidic is:', isAcidic);
+console.log('typeof isAcidic is:', typeof isAcidic);
+
+const headCount = 420;
+const isSparta = headCount === 300;
+
+console.log('value of isSparta is:', isSparta);
+console.log('typeof isSparta is:', typeof isSparta);
+
+let motto = fullName;
+motto = fullName += ' is the GOAT!';
```

Comment:
`+=` replaces `=` in this case. The `+=` operator concatenates or adds values, and then reassigns it to the variable. So this should just be `motto += ' is the GOAT!'`

Reply:
![Screenshot 2023-07-10 152924](https://github.com/dennis-pnguyen/c0723-code-solutions/assets/122412863/41572b05-6794-4305-9c38-f4122b5469b4)
![Screenshot 2023-07-10 152933](https://github.com/dennis-pnguyen/c0723-code-solutions/assets/122412863/fba05ce4-3056-4081-ba80-dda8fb94d5ed)

That's what I was missing, thanks Shawn!